modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx

```diff
@@ -128,6 +128,10 @@ describe('View Channel', () => {
 			'show'
 		);
 
+		expect(
+			screen.getByText('Unable to Delete Property')
+		).toBeInTheDocument();
```

Comment:
Hey @fmarcio, is it possible to validate that the modal text is present and not just the modal title?

See [here](https://liferay.atlassian.net/wiki/spaces/ENGAC/pages/2456355170/Test+Plan+LRAC-14082+Enable+Property+Deletion+from+DXP#Check-error-modal-message-and-hyperlink-on-deleting-property-that-has-sites-or-channels-synced---P3)